x-pack/plugins/fleet/server/routes/data_streams/handlers.ts

```diff
@@ -51,10 +56,22 @@ export const getListHandler: RequestHandler = async (context, request, response)
   };
 
   try {
+    const useMeteringApi = appContextService.getConfig()?.internal?.useMeteringApi;
```

Comment:
I'm wondering if we could use the metering API in stateful too, or is it only for serverless?

Reply:
Unfortunately it's only available in serverless